installer/project/components/configure_war.xml

```diff
@@ -267,11 +272,6 @@
           <key>security.server.hostname</key>
           <value>${hostname}</value>
         </propertiesFileSet>
-        <propertiesFileSet>
```

Comment:
See previous comment about removal of keys.

ripgrep shows `superUserEmail` is called by code in Aggregate, so it makes me nervous for a point release.

Reply:
I think ripgrep is matching `security.server.superUserUsername` which is another different key. We don't use `security.server.superUser` since the removal of google account credentials.